fixed typo

yyu opened the pull request:
`--event-handler` -> `--event-handlers`

dirk-thomas commented:
Thank you for the fix. Please target the default branch with this PR. Also the same mistake seems to be present in the `migration/catkin_tools.rst` file. Can you please address it there too.

yyu commented:
> Thank you for the fix. Please target the default branch with this PR. Also the same mistake seems to be present in the `migration/catkin_tools.rst` file. Can you please address it there too.

rebased to `master` and fixed them all. see #36 

thanks

dirk-thomas commented:
Superseded by #36.
